src/libraries/System.Security.Cryptography.Xml/tests/TestHelpers.cs

```diff
@@ -190,11 +190,50 @@ public static IEnumerable<SymmetricAlgorithmFactory> GetSymmetricAlgorithms(bool
     // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Suppression approved. Unit test dummy certificate.")]
```

Comment:
Probably want to copy/paste this line into the new PFX just to keep the scanner happy.

Reply:
Would I need to collapse this in to a single string like the other one to satisfy the scanner? I don't have access to credscan (at least that I what I choose to believe) and it seems inconsistently applied throughout the repo.

Reply:
> Would I need to collapse this in to a single string like the other one to satisfy the scanner? 

I hope not.  If not, then whenever there's the next "eep, this one got flagged" hopefully someone will see the attempted suppression and just make it happy.

> it seems inconsistently applied throughout the repo

Some of the places got nicely excluded by path.  Since this attribute is in this file I assume this wasn't one of them.